packages/p2p/src/components/buy-sell/no-ads/no-ads.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import { observer } from 'mobx-react-lite';
+import { useHistory } from 'react-router-dom';
 import { Button, Icon, Text } from '@deriv/components';
 import { Localize } from 'Components/i18next';
 import { useStores } from 'Stores';
 import './no-ads.scss';
+import { routes } from '@deriv/shared';
```

Comment:
can we change the order of import? move it above the css import maybe?

Reply:
Done :pray: